HouseBuilder/BigHouseBuilder.java

```diff
@@ -0,0 +1,41 @@
+package HouseBuilder;
+
+public class BigHouseBuilder implements HouseBuilder{
+
+	private House house;
+	
+	public BigHouseBuilder(){
+		house = new House();
+	}
+	
+	@Override
+	public void buildRooms() {
```

Comment:
I want to use this builder to get a house with 12 rooms. And then I want to use this same builder to get another house with 10 rooms. What do you need to change in order to accomplish this? Make the same modifications for the other build- methods too.